glide.yaml

```diff
@@ -5,6 +5,10 @@ import:
 - package: github.com/gorilla/mux
 - package: github.com/satori/go.uuid
 - package: github.com/spf13/cobra
+  version: f62e98d28ab7ad31d707ba837a966378465c7b57
```

Comment:
Not sure if this cobra version is needed. I initially put it in here as it is the exact version being used in k8s.

I think using --strip-vendor overwrites the version that glide pulls in directly.